docs/guide/generators.md

```diff
@@ -20,6 +20,18 @@ bin/rails generate component Example title content
       create    app/components/example_component.html.erb
 ```
 
+Organize components using namespaces. To generate a namespaced `Sections::ExampleComponent`:
```

Comment:
What are your thoughts on adding a subheader for this section?

Maybe something like: `## Generating namespaced components`?